### PR TITLE
[Merged by Bors] - 0.10.0 blog post: Android Support

### PR DESCRIPTION
**Pull request by mockersf**

### CODE REVIEW

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 1]
```diff
@@ -23,6 +23,22 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
...
+
+<div class="release-feature-authors">authors: @mockersf, @slyedoc</div>
+
+<div>
+<img src="./android%20emulator.png" alt="Android emulator running Bevy" width="50%" /><img src="./ios%20emulator.png" alt="iOS emulator running Bevy" width="50%" />
+</div>
+
+Bevy now runs out of the box on Android on more devices. This was unlocked by waiting for the [`Resumed`](https://docs.rs/winit/0.28/winit/event/enum.Event.html#variant.Resumed) event to create the window instead of doing it on startup, matching the [`onResume()`](https://developer.android.com/guide/components/activities/activity-lifecycle#onresume) callback on Android.
+
+To follow the recommandations on the [`Suspended`](https://docs.rs/winit/0.28/winit/event/enum.Event.html#variant.Suspended) event, Bevy will now exit on receiving that event. This is a temporary solution until Bevy is able to recreate rendering resources when being resumed.
```
Comment (kristoff3r) [inline content/news/2023-03-04-bevy-0.10/index.md:36]:
```suggestion
To follow the recommendations on the [`Suspended`](https://docs.rs/winit/0.28/winit/event/enum.Event.html#variant.Suspended) event, Bevy will now exit on receiving that event. This is a temporary solution until Bevy is able to recreate rendering resources when being resumed.
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 2]
```diff
@@ -23,6 +23,22 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 Description here.
 
+## Android Support
+
+<div class="release-feature-authors">authors: @mockersf, @slyedoc</div>
+
+<div>
+<img src="./android%20emulator.png" alt="Android emulator running Bevy" width="50%" /><img src="./ios%20emulator.png" alt="iOS emulator running Bevy" width="50%" />
```
Comment (cart) [inline content/news/2023-03-04-bevy-0.10/index.md:31]: We might want to sort out a different android screenshot given the stretching.  Maybe better to snag a screenshot from hardware itself. Unless things have changed I don't think this is a problem on real hardware?

Regardless, that can wait until a later pass.

**Review by cart — APPROVED** [review]

File: content/news/2023-03-04-bevy-0.10/index.md (hunk 2 above)
Comment (mockersf) [reply content/news/2023-03-04-bevy-0.10/index.md:31]: I only tried on landscape mode on hardware and it wasn't an issue... but I currently don't have android hardware available

Comment (mockersf) [reply content/news/2023-03-04-bevy-0.10/index.md]: updated the screenshots to be in landscape and they don't have stretching issue

**Pull request closed without merging**